contracts/contracts/token/WOETH.sol

```diff
@@ -49,7 +81,161 @@ contract WOETH is ERC4626, Governable, Initializable {
         external
         onlyGovernor
     {
+        //@dev TODO: we could implement a feature where if anyone sends OETH direclty to
+        // the contract, that we can let the governor transfer the excess of the token.
         require(asset_ != address(asset()), "Cannot collect OETH");
         IERC20(asset_).safeTransfer(governor(), amount_);
     }
+
+    /**
+     * @dev This function converts requested OETH token amount to its underlying OETH
+     * credits value that is stored internally in OETH.sol and is required in order to
+     * be able to rebase.
+     *
+     * @param oethAmount Amount of OETH to be converted to OETH credits
+     * @return amount of OETH credits the OETH amount corresponds to
+     */
+    function _oethToOethCreditsHighres(uint256 oethAmount)
```

Comment:
Perhaps this would clearer if this method was named `_creditsPerAsset`, and the state variable `oethCreditsHighres` changed to `assetCredits`. 

In this way all the 4626 code gets to use clear variable names, and the OETH related stuff ends up just inside `_creditsPerAsset`.

Reply:
good point thanks, renamed